Misc/NEWS.d/next/Core and Builtins/2020-06-12-00-12-28.bpo-40950.tzMy7m.rst

```diff
@@ -0,0 +1 @@
+Apply :pep:`3121` to :mod:`nis`. Patch by Dong-hee Na.
```

Comment:
I'm unable to recall *all* PEPs. Maybe explain keep the PEP reference, but explain that the change adds a state of the nis module?

```suggestion
Add a state to the :mod:`nis` module (:pep:`3121`). Patch by Dong-hee Na.
```